PendingReleaseNotes.md

```diff
@@ -26,6 +28,9 @@
   - Namespaces: The example namespaces are now backend-specific. Instead of `rook-system` and `rook`, you will see `rook-ceph-system` and `rook-ceph`.
   - Volume plugins: The dynamic provisioner and flex driver are now based on `ceph.rook.io` instead of `rook.io`
 - Ceph container images now use CentOS 7 as a base
+- It is required to set a `mon.count`, see [`mon` spec](Documentation/ceph-cluster-crd.md#mon-settings).
```

Comment:
if we set the default during upgrade, seems like we wouldn't need this breaking change